commands-v2/ramsete/subsystems/drivetrain.py

```diff
@@ -0,0 +1,118 @@
+from commands2 import SubsystemBase
+
+from wpilib import SpeedControllerGroup, PWMVictorSPX, Encoder
+from wpilib.drive import DifferentialDrive
+from wpilib.interfaces import GenericHID
+
+from wpimath.geometry import Pose2d, Rotation2d
+from wpimath.kinematics import DifferentialDriveOdometry, DifferentialDriveWheelSpeeds
+
+from constants import constant
+
+
+class Drivetrain(SubsystemBase):
+    def __init__(self, controller: GenericHID):
+
+        super().__init__()
+
+        frontLeftMotor, backLeftMotor = (
+            PWMVictorSPX(constant.frontLeftMotorID),
+            PWMVictorSPX(constant.backLeftMotorID),
+        )
+        frontRightMotor, backRightMotor = (
+            PWMVictorSPX(constant.frontRightMotorID),
+            PWMVictorSPX(constant.backRightMotorID),
+        )
+
+        self.leftMotors = SpeedControllerGroup(frontLeftMotor, backLeftMotor)
+
+        self.rightMotors = SpeedControllerGroup(frontRightMotor, backRightMotor)
+
+        self.drivetrain = DifferentialDrive(self.leftMotors, self.rightMotors)
+        self.drivetrain.setRightSideInverted(True)
+
+        self.leftEncoder = Encoder(
+            constant.leftEncoderPorts[0],
+            constant.leftEncoderPorts[1],
+            constant.leftEncoderReversed,
+        )
+
+        self.leftEncoder.setDistancePerPulse(constant.encoderDistancePerPulse)
+
+        self.rightEncoder = Encoder(
+            constant.rightEncoderPorts[0],
+            constant.rightEncoderPorts[1],
+            constant.rightEncoderReversed,
+        )
+
+        self.rightEncoder.setDistancePerPulse(constant.encoderDistancePerPulse)
+
+        self.gyro = constant.gyroObject
+        self.gyro.calibrate()
+
+        self.odometry = DifferentialDriveOdometry(self.gyro.getRotation2d())
+
+        self.controller = controller
+
+        self.resetEncoders()
+
+    def periodic(self):
+        print(self.odometry.getPose())
```

Comment:
Print statement shouldn't be here?

Reply:
Yeah that too. From debugging earlier, will remove